packages/gatsby-source-sensenet/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { defaultRepositoryConfiguration, ODataUrlBuilder } from '@sensenet/client-core'
+import { SourceNodesArgs } from 'gatsby'
+import fetch from 'node-fetch'
+import { PluginConfig } from './gatsby-node'
+
+export const snPrefix = 'sensenet'
+
+export const createTreeNode = async (
+  parentNode: any,
+  content: any,
```

Comment:
GenericContent?

Reply:
I think it cannot be GenericContent cause it can be any content. 
Otherwise it cause type-error
![image](https://user-images.githubusercontent.com/22347393/120806706-7b7be880-c547-11eb-95f5-5195ab20c9a7.png)